src/AppBundle/Entity/Receipt.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Entity\Repository\ReceiptRepository")
+ * @ORM\Table(name="receipt")
+ */
+class Receipt
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="User", inversedBy="receipts", cascade={"persist"})
+     */
+    private $user;
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $submitDate;
+
+    /**
+     * @ORM\Column(name="picture_path", type="string", length=45, nullable=true)
+     */
+    private $picturePath;
+
+    /**
+     * @ORM\Column(type="string")
+     * @Assert\NotBlank(message="Dette feltet kan ikke være tomt.")
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="float")
+     * @Assert\NotBlank(message="Dette feltet kan ikke være tomt.")
+     */
+    private $sum;
+
+    /**
+     * @var bool
+     * @ORM\Column(type="boolean")
+     */
+    private $active;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $visualId;
+
+    /**
+     * Receipt constructor.
+     */
+    public function __construct()
+    {
+        $this->active = true;
+        $this->submitDate = new \DateTime();
+        $this->visualId = substr(md5(uniqid()), -7);
```

Comment:
Ok, enig i at det blir mye kompleksitet for lite gevinst. Men jeg synes ikke helt om å bytte ut visualIDene med et script, hvis vi skulle trenge for det. Da vil jo alle referanser til tidligere visualIDer i mailer osv. ikke stemme lenger. Men med den alternative løsningen blir det veldig mye lettere. Da slipper vi å vente på at IDen skal autogenereres og trenger derfor ikke gå via `receiptCreated` eventen. Tror jeg går for den løsningen.